src/Manager.js

```diff
@@ -0,0 +1,29 @@
+class Manager {
+  constructor() {
+    this.cohorts = []
+  }
+
+  addCohort(cohort) {
+    this.cohorts.push(cohort)
+  }
+
+  removeCohortByName(name) {
+    const originalLength = this.cohorts.length
+    this.cohorts = this.cohorts.filter((cohort) => cohort.CohortName !== name)
+
+    if (this.cohorts.length === originalLength) {
+      throw new Error('Cohort does not exist')
+    }
```

Comment:
You've already implemented code that does this so you're repeating yourself here. You could remove this and just run `this.findCohortByName(name)`